vmdb/app/controllers/mixins/vm_show_mixin.rb

```diff
@@ -111,16 +111,16 @@ def show_record(id = nil)
       return
     end
 
-    if @record.class.base_model.to_s == "MiqTemplate"
-      rec_cls = @record.class.base_model.to_s.underscore
-    else
-      rec_cls = "vm"
-    end
     case @display
     when "download_pdf", "main", "summary_only"
+      if @record.class.base_model.to_s == "MiqTemplate"
```

Comment:
if we use my suggestion above and the ternary operator, we can simplify the whole if/else construct to `@button_group = @record.kind_of?(MiqTemplate) ? 'miq_template' : 'vm'`